packages/cryptography/test_cryptography.py

```diff
@@ -18,149 +18,6 @@ def test_cryptography():
     assert f1.decrypt(f.encrypt(b"abc")) == b"abc"
 
 
-@run_in_pyodide(packages=["cryptography", "pytest"])
```

Comment:
These are no longer relevant or do they fail?

Reply:
I think these test features that were removed between 3.4.8 and 36.0.2, so they fail with name errors. Maybe we should ask the cryptography maintainers for more info on what to test.